db/src/bootstrap.rs

```diff
@@ -195,14 +197,20 @@ fn symbolic_schema_to_triples(ident_map: &IdentMap, symbolic_schema: &Value) ->
                             // bootstrap symbolic schema, or by representing the initial bootstrap
                             // schema directly as Rust data.
                             let typed_value = match TypedValue::from_edn_value(value) {
-                                Some(TypedValue::Keyword(ref s)) => TypedValue::Ref(*ident_map.get(s).ok_or(ErrorKind::UnrecognizedIdent(s.clone()))?),
+                                Some(TypedValue::Keyword(ref s)) => {
+                                    let entid = to_namespaced_keyword(s)
```

Comment:
This is fine, but it would be more consistent if you further `map` all the way into the `TypedValue::Ref`.